crawler4j-commons/src/main/java/edu/uci/ics/crawler4j/crawler/DynamicCrawlConfig.java

```diff
@@ -0,0 +1,101 @@
+package edu.uci.ics.crawler4j.crawler;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.util.Collections;
+import java.util.List;
+
+import static edu.uci.ics.crawler4j.crawler.DynamicCrawlConfig.WebDriverType.firefox;
```

Comment:
Don't think we need that static import? We can just reference the full qualifier in line 20

Reply:
Right, fixed.